.travis.yml

```diff
@@ -1,10 +1,9 @@
 sudo: false
 language: python
 python:
-- 2.7
 - 3.6
-- 3.7
```

Comment:
Why do we drop 3.7, but maintain 3.6, 3.8 and 3.9?

Reply:
Based on https://github.com/OnroerendErfgoed/skosprovider_getty/pull/87#pullrequestreview-815061712

Reply:
That explains why we kept compatibility with 3.6. This is an OSS library that should follow it's own logic. If we maintain 3.6 compatibility for longer, then we should also maintain 3.7 compatibility. This takes up a bit more travis resources, but especially for a small library such as this, it's minimal. For applications such as atramhasis (and especially internal stuff) it makes more sense to drop versions to reduce the load on travis.

Reply:
I agree